.github/workflows/push.yml

```diff
@@ -0,0 +1,27 @@
+name: PR Test
+
+on:
+  pull_request:
+    branches: [ master ]
+
```

Comment:
Also run on merge?

```
on:
  pull_request:
    branches: [ master ]
  push:
    branches: [ master ]
```

Reply:
Nope just PR, that's how we do it on all the other things.  Saves resources.

Reply:
Running it again on master will catch a bad merge. Build minutes are free on public.